apps/web/src/pages/integrations/components/LimitBar.tsx

```diff
@@ -1,85 +1,103 @@
-import { useMemo } from 'react';
-import { ChannelTypeEnum } from '@novu/shared';
-import { Stack } from '@mantine/core';
-import { colors, Tooltip } from '../../../design-system';
-import { useIntegrationLimit } from '../../../hooks';
 import styled from '@emotion/styled/macro';
+import { Stack, useMantineColorScheme } from '@mantine/core';
+import { ChannelTypeEnum } from '@novu/shared';
+import { useMemo } from 'react';
 import { Link } from 'react-router-dom';
+import { When } from '../../../components/utils/When';
+import { colors, Text } from '../../../design-system';
+
+const WARNING_LIMIT = {
+  [ChannelTypeEnum.EMAIL]: 50,
+  [ChannelTypeEnum.SMS]: 5,
+};
 
 export const LimitBar = ({
   withLink = false,
   channel = ChannelTypeEnum.EMAIL,
-  label = null,
+  showDescription,
+  limit,
+  count,
+  loading,
 }: {
   withLink?: boolean;
   channel?: ChannelTypeEnum;
-  label?: string | null;
+  showDescription?: boolean;
+  limit: number;
+  count: number;
+  loading: boolean;
 }) => {
-  const {
-    data: { limit, count },
-    loading,
-    isLimitFetchingEnabled,
-  } = useIntegrationLimit(channel);
-
   if (channel !== ChannelTypeEnum.EMAIL && channel !== ChannelTypeEnum.SMS) {
     return null;
   }
 
-  if (loading || !isLimitFetchingEnabled) {
+  if (loading) {
     return null;
   }
 
   if (withLink) {
     return (
       <Link to="/integrations">
-        <LimitBarBase limit={limit} count={count} channel={channel} label={label} />
+        <LimitBarBase limit={limit} count={count} channel={channel} showDescription={showDescription} />
       </Link>
     );
   }
 
-  return <LimitBarBase limit={limit} count={count} channel={channel} label={label} />;
+  return <LimitBarBase limit={limit} count={count} channel={channel} showDescription={showDescription} />;
 };
 
 const LimitBarBase = ({
   limit,
   count,
   channel = ChannelTypeEnum.EMAIL,
-  label = null,
+  showDescription,
 }: {
   limit: number;
   count: number;
   channel?: ChannelTypeEnum;
-  label?: string | null;
+  showDescription?: boolean;
 }) => {
   const unit = useMemo<string>(() => {
-    return channel === ChannelTypeEnum.EMAIL ? 'emails' : 'sms';
+    return channel === ChannelTypeEnum.EMAIL ? 'emails' : 'messages';
```

Comment:
I think since we removed the Tooltip component this `useMemo` is not needed anymore as there won't be child components rerenders and we can just calculate the string.
CC: @LetItRock am I wrong with this?